cartaya/card-2/index.html

```diff
@@ -8,18 +8,30 @@
     <title>card-2</title>
 </head>
 <body>
-    <div class="card-group_container">
-        <div class="card">
-            <img class="card_icon" src="./resources/icon/white.svg" alt="thunderbolt icon">
-            <p class="card_text">Your Top-up is received Al Instante</p>
+    <div class="card-group">
+        <div class="card card_one">
```

Comment:
You don't need create a class for each card. You can apply to the class card styles by default for the cards, after put class to the modify elements